src/cpu.c

```diff
@@ -452,6 +456,28 @@ static void cpu_commit_one (int cpu_num, /* {{{ */
 	}
 } /* }}} void cpu_commit_one */
 
+/* Commits the number of cores */
+static void cpu_commit_num_cpu (gauge_t num_cpu) /* {{{ */
+{
+//// static void submit_value (int cpu_num, int cpu_state, const char *type, value_t value)
+	value_t values[1];
+	value_list_t vl = VALUE_LIST_INIT;
+
+	/// memcpy(&values[0], &value, sizeof(value));
+	values[0].gauge = num_cpu;
+
+	vl.values = values;
+	vl.values_len = 1;
+
+	sstrncpy (vl.host, hostname_g, sizeof (vl.host));
+	sstrncpy (vl.plugin, "cpu", sizeof (vl.plugin));
+	sstrncpy (vl.type, "gauge", sizeof (vl.type));
```

Comment:
I suggest to use type "count" and an empty type instance for this metric, i.e. "<host>/cpu/count".